pkgs/by-name/hy/hyprfreeze/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  jq,
+  procps,
+  psmisc,
+  libnotify,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hyprfreeze";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Zerodya";
+    repo = pname;
```

```suggestion
    repo = "hyprfreeze";
```
Usually this is hardcoded

---

Thanks, idk how I missed that.